### ISSUE TITLE
Webpack - How to add Javascript Library (NPM / Bower / file), for example slick.js

### ISSUE THREAD

**Issue by retani** [MozaikAgency/wp-theme-bootstrap#273]
Hi there,

I checked many starter themes that include a build system and chose this one. I am quit happy with it ... EXCEPT I cannot figure out how to add a javascript library. Maybe you can  point out the workflow? I have followed the jQuery in the readme, but without success. I have also read about webpack and check the gulp files.

At the moment, I want to add "slick" for sliders, and as I see, the lazyLoad is already prepared, so it's not an uncommon thing to do.

When I copy slick.js to assets/vendor/ and include it to assets/main.js with "import './vendor/slick'", I run into problems, as the build system adds code on top of slick.js. I cannot even figure out how it does that. Line 3 of that code ("var _typeof2 = __webpack_require__" ... )then produces an error: 

slick.js:3 Uncaught Error: Cannot find module "babel-runtime/helpers/typeof"

I tried to add it with "bower install slick --save", but then I am already stuck again. How do I link the css and js files?

Long story short: Could someone explain to me (and maybe others) how to add a library such as slick the best way?

**Comment by takosuke** [comment]
bump to this. Also followed the (first) jQuery example without success. I haven't had the time to learn enough about gulp and webpack to get this going so I assume I'm missing something, but yeah just following the readme example doesn't seem to do it.

**Comment by retani** [comment]
Well, one important thing that I found out is that slick on npm is "slick-carousel", not "slick" (!): 
`npm install slick-carousel` or `bower install slick-carousel`. That should already make it ready to use.
